sdk/ai/azure-ai-inference/samples/sample_chat_completions_with_audio_url.py

```diff
@@ -0,0 +1,87 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to get a chat completions response from
+    the service using a synchronous client. The sample shows how to use a
+    url pointer to an audio file in the input chat messages.
+    This sample will only work on AI models that support audio input.
+    Only these AI models accept the array form of `content` in the
+    `UserMessage`, as shown here.
+
+    This sample assumes the AI model is hosted on a Serverless API or
+    Managed Compute endpoint. For GitHub Models or Azure OpenAI endpoints,
+    the client constructor needs to be modified. See package documentation:
+    https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-inference/README.md#key-concepts
+
+USAGE:
+    python sample_chat_completions_with_audio_data.py
+
+    Set these two or three environment variables before running the sample:
+    1) AZURE_AI_CHAT_ENDPOINT - Your endpoint URL, in the form
+        https://<your-deployment-name>.<your-azure-region>.models.ai.azure.com
+        where `your-deployment-name` is your unique AI Model deployment name, and
+        `your-azure-region` is the Azure region where your model is deployed.
+    2) AZURE_AI_CHAT_KEY - Your model key. Keep it secret.
+    3) AZURE_AI_CHAT_DEPLOYMENT_NAME - Optional. The value for the HTTP
+        request header `azureml-model-deployment`.
+"""
+
+
+def sample_chat_completions_with_audio_url():
+    import os
+    from azure.ai.inference import ChatCompletionsClient
+    from azure.ai.inference.models import (
+        SystemMessage,
+        UserMessage,
+        TextContentItem,
+        AudioUrlContentItem,
+        InputAudioUrl,
+    )
+    from azure.core.credentials import AzureKeyCredential
+
+    try:
+        endpoint = os.environ["AZURE_AI_CHAT_ENDPOINT"]
+        key = os.environ["AZURE_AI_CHAT_KEY"]
+    except KeyError:
+        print("Missing environment variable 'AZURE_AI_CHAT_ENDPOINT' or 'AZURE_AI_CHAT_KEY'")
+        print("Set them before running this sample.")
+        exit()
+
+    try:
+        model_deployment = os.environ["AZURE_AI_CHAT_DEPLOYMENT_NAME"]
+    except KeyError:
+        print("Could not read optional environment variable `AZURE_AI_CHAT_DEPLOYMENT_NAME`.")
+        print("No specific model target will not be set.")
+        model_deployment = None
+
+    audio_url = (
+        "https://raw.githubusercontent.com/Azure/azure-sdk-for-python/refs/tags/azure-ai-inference_1.0.0b9/"
+        + "sdk/ai/azure-ai-inference/samples/hello_how_are_you.mp3"
+    )
```

Comment:
Where did you get the above link? I get this one, and it does not show "1.0.0b9":
https://github.com/Azure/azure-sdk-for-python/raw/refs/heads/main/sdk/ai/azure-ai-inference/samples/hello_how_are_you.mp3